src/app/stores/Store.js

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable no-underscore-dangle */
 import configureStore from './configureStore';
+import initialState from './InitialState';
 
 const preloadedState = global && global.window && global.window.__PRELOADED_STATE__;
 if (preloadedState) {
   delete global.window.__PRELOADED_STATE__;
 }
 
-const store = configureStore(preloadedState);
+const store = configureStore(preloadedState || initialState);
```

Comment:
How come you added `initialState` here?

Reply:
This may be leftover from previous work. Will check if it is really needed and remove if not.